dbt/sagerx/models/intermediate/rxnorm/int_mthspl_products_to_inactive_ingredients.sql

```diff
@@ -1,17 +1,14 @@
 -- int_mthspl_products_to_inactive_ingredients.sql
+{{ config(materialized='table') }}
```

Comment:
I need to remove this - it was only to make testing quicker.

Reply:
shouldnt this materialize anyway as a table since we have materialized as table set for intermediates?

Reply:
Yeah I think you're right.  I might have been working off of a branch that didn't have that setting.  I might clean this up just for the sake of it.